README.md

```diff
@@ -119,10 +138,9 @@ The last operation of the else part is to assign the current degree value to the
 
 The same procedure is used for the right motor.
 
+## Displaying the progress
 
-### Quang Thanh Lai (Route Creation &  Driving Controls)
-TODO
-
+Once the map has been converted from the input string to a 2D array and the start and end points have been processed by the system, this information is printed to the display. This view is updated when the robot starts driving to its first target. Once the robot starts moving, the route is only updated when the robot makes a turn. This design choice was made to keep the display logic out of the driving functions. After each update, you can also see the robot's current position and the destination it's heading towards. The same behaviour can be observed when the robot navigates back to its starting position, or the robot travels to the second target.
```

Comment:
- ... when the robot navigates back to its starting position, or 'when it' travels to the second target.

Reply:
Fixed!